README.md

```diff
@@ -1,71 +1,139 @@
-# Kurtosis-Charon setup
+# Kurtosis-Charon
+
+## Overview
+
+This project leverages [kurtosis](https://docs.kurtosis.com) to run a local setup of ethereum and beacon chains with DVT.
+It does not connect to real beacon chains and instead it creates a preconfigured local environment, similar to [Ganache](https://archive.trufflesuite.com/ganache/).
+Once started, you will have several instances of execution clients, consensus clients, charon and validator clients running altogether.
+What's important to us: kurtosis takes care of configuring and activating validator keys, we only need to import the keys to run DVT.
+
+Kurtosis supports all of the existing vendors: lighthouse, nimbus, prysm, teku and lodestar for both BN and VC roles.
```

Comment:
We should link that kurtosis table that details compatibility between BNs and VCs. Not all combinations are supported under kurtosis, even without DVT.

Reply:
Added